metricbeat/Makefile

```diff
@@ -90,6 +90,6 @@ run-module:
 
 .PHONY: test-module
 test-module: ## @testing Tests the given module. Needs $MODULE as param an run-module must be started first.
-test-module: python-env
+test-module: python-env fields metricbeat.test
```

Comment:
Reminds me of https://github.com/elastic/beats/pull/4961 which strangely fails on travis. I wonder if it should be more then just `fields` but `update`.

Reply:
Ok, using `update` instead.